### PR TITLE
[i18n] Updated all Romanian Translations and fix english

### PR DESCRIPTION
**Pull request by ImGelu**
Since the Romanian translations were pretty bad and very outdated, I decided to translate all the strings from scratch.

Also, I made some small grammar/typo fixes in the English source strings. Hope that's ok!

### CODE REVIEW

**Comment by ImGelu** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by masterbd — APPROVED** [review]

**Comment by mcmire** [comment]
Hey, thanks for the PR. We have a lot of new translations coming in via #13206, so I'll wait until that's merged first before looking at this.

**Comment by brad-decker** [comment]
@ImGelu this looks good, but I want to get one of our wordsmiths to take a look at the changes to en messages. Could you do me a favor and rebase to see if any of your changes have already been applied then i'll get someone to review. :)

File: app/_locales/en/messages.json
Comment (mcmire) [inline app/_locales/en/messages.json:1521]: Hmm, looks like there's still a typo here. This should be "a user's".

**Review by mcmire — COMMENTED** [review]
I think these are reasonable changes — just had a quick note on one of them!

File: app/_locales/en/messages.json
Comment (ImGelu) [reply app/_locales/en/messages.json:1521]: Nope, "an" is used before a word that starts with a vowel :) https://owl.purdue.edu/owl/general_writing/grammar/articles_a_versus_an.html

**Comment by ImGelu** [comment]
> @ImGelu this looks good, but I want to get one of our wordsmiths to take a look at the changes to en messages. Could you do me a favor and rebase to see if any of your changes have already been applied then i'll get someone to review. :)

Sure, I'll do it in the upcoming days :)

File: app/_locales/en/messages.json
Comment (ImGelu) [reply app/_locales/en/messages.json:1521]: Noted. Will change.

**Comment by brad-decker** [comment]
@ImGelu I am going to close this PR due to inactivity, but we encourage you to make the requested changes, rebase and reopen when you're ready for it to be reconsidered. Thanks for your submission, we appreciate it!

**Pull request closed without merging**
